### PR TITLE
Apger first change to content ./endpoint/winword_spawning_powershell.yml

### PR DESCRIPTION
**Pull request by apger**
I don't use git that often outside of my own project but here is my first attempt at contributing to the greater Splunk good.  please advise and I get to work if attempt is inline with your process.

### CODE REVIEW

File: detections/endpoint/winword_spawning_powershell.yml [hunk 1]
```diff
@@ -52,3 +52,24 @@ tags:
...
   - parent_process_id
   security_domain: endpoint
+  impact: 5000
+  confidence: 1
+  # (impact * confidence)/100
+  risk_score: 50
+  context:
+  - Source:Endpoint
+  - Stage:Execution
+  message: $parent_process_name$ on $dest$ by $user$ launched the following powershell process: $process_name$ which is very common in phishing attacks.
```
Comment (patel-bhavin) [inline detections/endpoint/winword_spawning_powershell.yml]: Hey @apger : looks like our validation is failing on the `value`of the message, can you change `process:` to `process-`

Comment (josehelps) [reply detections/endpoint/winword_spawning_powershell.yml]: alternatively tie that around quotes so it gets treated as a string :smile:

Comment (apger) [reply detections/endpoint/winword_spawning_powershell.yml]: Fixed with Bhavin

**Comment by apger** [comment]
Changes made - new pull request coming.

**Pull request closed without merging**